### PR TITLE
feat: ability to filter blogs by tags

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
Yet to figure out what is causing the issue with the png on the scale-up effect of blog thumbnails

**Comment by Nikhil-Ladha** [comment]
Everything looks good. Just a small update if the tags like `open-source` and some similar one's can be capitalized like others, it would look better.

File: src/sections/Navigation/utility/data.js [hunk 1]
```diff
@@ -54,10 +54,6 @@ const Data = {
                 {
                     name: "Blog Grid",
                     path: "/blog-grid"
-                },
-                {
-                    name: "Blog List",
-                    path: "/blog-list"
```
Comment (leecalcote) [inline src/sections/Navigation/utility/data.js:60]: Don't we want to keep this view as well?

Comment (Nikhil-Ladha) [reply src/sections/Navigation/utility/data.js:60]: Here, we are using that view for the filtered blogs. But, I had a different thought for that view. And will discuss it in the meeting this Thursday. Then, we can decide on that part.

Comment (leecalcote) [reply src/sections/Navigation/utility/data.js:60]: ok, @Nikhil-Ladha. Are we good to move forward in the meantime?

Comment (leecalcote) [reply src/sections/Navigation/utility/data.js:60]: I have added to the meeting minutes so that we don't forget this discussion.

Comment (Nikhil-Ladha) [reply src/sections/Navigation/utility/data.js:60]: I think we can hold it for a day. And, then we can move forward with what to do.

**Comment by Tanuj22** [comment]
@leecalcote @Nikhil-Ladha commented out the `blog-list` page.

**Pull request merged**